ts/features/design-system/core/DSIcons.tsx

```diff
@@ -179,6 +186,29 @@ export const DSIcons = () => (
             />
           ))}
         </View>
+        <H2
+          color={theme["textHeading-default"]}
+          weight={"SemiBold"}
+          style={{ marginBottom: 12 }}
+        >
+          System
+        </H2>
+        <View style={styles.itemsWrapper}>
+          {Object.entries(IOSystemIcons).map(([iconItemName]) => (
+            <DSIconViewerBox
+              key={iconItemName}
+              name={iconItemName}
+              size="large"
```

Comment:
Just a small suggestion to display the icon at a smaller size on the DS screen.
```suggestion
              size="medium"
```